scheduler/pmc-processor-task/src/test/resources/file1.txt

```diff
@@ -0,0 +1 @@
+that is the content
```

Comment:
couldn't we just create these test files with the temporary folder extension?

Reply:
yes no point to add this a resource